runtime/libpgmath/CMakeLists.txt

```diff
@@ -117,3 +122,11 @@ include(LibmathUtils)
 # Setup Source Code And Tests
 add_subdirectory(lib)
 add_subdirectory(test)
+
+if(0)
```

Comment:
Is this left-over debugging code?

Reply:
Yes, the developer here wanted to leave it in for the time being.